lib/dropbox_api/endpoints/files/list_folder.rb

```diff
@@ -21,19 +21,25 @@ class ListFolder < DropboxApi::Endpoints::Rpc
     #   returned for deleted file or folder, otherwise
     #   {DropboxApi::Errors::NotFoundError}
     #   will be raised. The default for this field is `false`.
+    # @option options limit [Numeric] If present, will specify max number of
+    #   results per request (Note:
+    #   {https://www.dropbox.com/developers/documentation/http/documentation#files-list_folder Dropbox docs} indicate
+    #   this is "approximate", and more may be returned)
     add_endpoint :list_folder do |path, options = {}|
       validate_options([
         :recursive,
         :include_media_info,
         :include_deleted,
         :shared_link,
-        :include_has_explicit_shared_members
+        :include_has_explicit_shared_members,
+        :limit
       ], options)
       options[:recursive] ||= false
       options[:include_media_info] ||= false
       options[:include_deleted] ||= false
       options[:shared_link] = build_shared_link_param(options[:shared_link]) if options[:shared_link]
-
+      options[:limit] = options[:limit] if options[:limit]
+      
```

Comment:
What does this line do?

Reply:
oops - not much.  that should be removed.  sorry about that.  I can PR with a removal, if you like.

Reply:
Nevermind, I'll remove it when I get a chance.

Thanks for your contribution!